dev-perl/Quota/Quota-1.7.2.ebuild

```diff
@@ -24,8 +25,12 @@ DIST_TEST=skip
 
 src_prepare() {
 	default
-	export mymake="OPTIMIZE=$($(tc-getPKG_CONFIG) --cflags libtirpc)"
+	export mymake="INC=$($(tc-getPKG_CONFIG) --cflags libtirpc) OTHERLDFLAGS=$($(tc-getPKG_CONFIG) --libs libtirpc)"
 
 	# disable AFS completely for now, need somebody who can really test it
 	sed -i -e 's|-d "/afs"|0|' Makefile.PL || die "sed failed"
 }
+
+src_test() {
+	perl -Mblib -e 'use Quota' || die "Quota module not usable"
```

Comment:
Not quite sure this is the right way to invoke the Perl interpreter in the src_test phase. @kentfredric what's your take on this?

Reply:
That strategy is only kinda acceptable when the module in question has no tests of its own. I've been doing a bit of that recently because it beats having no tests ( or having no runnable tests )

Reply:
The included tests by the module itself are unfortunately interactive only.
```pl
# Tests need real FS access/root permissions and are interactive
DIST_TEST=skip
```